### ISSUE TITLE
How to fix "Concurrency is only available in iOS 15.0.0 or newer" error Flutter

### ISSUE THREAD

**Issue by Marvioso** [flutter/flutter#106976]
I recently updated my Flutter SDK to 3.0.4, and my project would no longer build when trying to launch on the iOS Simulator.

I got the following errors and implemented the solutions below each error:

``
The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99.
``
Solution that has worked in the past but fails now since this is also included in the error message: https://stackoverflow.com/questions/61823044/flutter-the-ios-simulator-deployment-target-iphoneos-deployment-target-is-se 

and 
``
error: 'Flutter/Flutter.h' file not found
``

Solution when I got above error, worked in the past, not now: https://stackoverflow.com/questions/64973346/error-flutter-flutter-h-file-not-found-when-flutter-run-on-ios (basically deleting and rebuilding ios folder)

I then tried to re-implement this (https://codewithandrea.com/tips/speed-up-cloud-firestore-xcode-builds/) after rebuilding the ios folder (requires a new podfile) in the solution above as the builds took forever to fail, and it worked well before the Flutter update. I used firestore version 9.2.0 since that is what was in my podfile.lock file.

This is when I started getting the swift compiler error I cannot solve, shown below. Other projects are running with the flutter update and on this simulator. Even after reversing the solution from the "code with Andrea" link above, I still get this same error. 

pod install, pod update, flutter pub cache repair, flutter clean, flutter pub get, pod install --repo-update, none of these solve the error. 

Please help.

Error message (it wouldn't let me post the whole thing, character limits):
``
Running "flutter pub get" in project_name...
Launching lib/main.dart on iPhone 12 in debug mode...
Xcode build done.                                           300.1s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **
Xcode's output:
↳
    Writing result bundle at path:
    	/var/folders/5m/btwjgwbs3zz4xt0v29h78lsm0000gn/T/flutter_tools.GaL813/flutter_ios_build_temp_dirDysItT/temporary_xcresult_bundle
/Users/user1/development/tools/flutter/.pub-cache/hosted/pub.dartlang.org/location-4.4.0/ios/Classes/LocationPlugin.m:176:13: warning: 'UIAlertView' is deprecated: first deprecated in iOS 9.0 - UIAlertView is deprecated. Use UIAlertController with a preferredStyle of UIAlertControllerStyleAlert instead [-Wdeprecated-declarations]
                UIAlertView *alert = [[UIAlertView alloc] initWithTitle:@"Location is Disabled"
                ^
    In module 'UIKit' imported from /Users/projectPath/ios/Pods/Target Support Files/location/location-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIAlertView.h:27:12: note: 'UIAlertView' has been explicitly marked deprecated here
    @interface UIAlertView : UIView
               ^
/Users/user1/development/tools/flutter/.pub-cache/hosted/pub.dartlang.org/location-4.4.0/ios/Classes/LocationPlugin.m:176:48: warning: 'UIAlertView' is deprecated: first deprecated in iOS 9.0 - UIAlertView is deprecated. Use UIAlertController with a preferredStyle of UIAlertControllerStyleAlert instead [-Wdeprecated-declarations]
                UIAlertView *alert = [[UIAlertView alloc] initWithTitle:@"Location is Disabled"
                                                   ^
    In module 'UIKit' imported from /Users/projectPath/ios/Pods/Target Support Files/location/location-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIAlertView.h:27:12: note: 'UIAlertView' has been explicitly marked deprecated here
    @interface UIAlertView : UIView
               ^
    2 warnings generated.
/Users/user1/development/tools/flutter/.pub-cache/hosted/pub.dartlang.org/location-4.4.0/ios/Classes/LocationPlugin.m:176:13: warning: 'UIAlertView' is deprecated: first deprecated in iOS 9.0 - UIAlertView is deprecated. Use UIAlertController with a preferredStyle of UIAlertControllerStyleAlert instead [-Wdeprecated-declarations]
                UIAlertView *alert = [[UIAlertView alloc] initWithTitle:@"Location is Disabled"
                ^
    In module 'UIKit' imported from /Users/projectPath/ios/Pods/Target Support Files/location/location-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIAlertView.h:27:12: note: 'UIAlertView' has been explicitly marked deprecated here
    @interface UIAlertView : UIView
               ^
/Users/user1/development/tools/flutter/.pub-cache/hosted/pub.dartlang.org/location-4.4.0/ios/Classes/LocationPlugin.m:176:48: warning: 'UIAlertView' is deprecated: first deprecated in iOS 9.0 - UIAlertView is deprecated. Use UIAlertController with a preferredStyle of UIAlertControllerStyleAlert instead [-Wdeprecated-declarations]
                UIAlertView *alert = [[UIAlertView alloc] initWithTitle:@"Location is Disabled"
                                                   ^
    In module 'UIKit' imported from /Users/projectPath/ios/Pods/Target Support Files/location/location-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIAlertView.h:27:12: note: 'UIAlertView' has been explicitly marked deprecated here
    @interface UIAlertView : UIView
               ^
    2 warnings generated.
    1 warning generated.
    error: the following command failed with exit code 0 but produced no further output
CompileC /Users/user1/Library/Developer/Xcode/DerivedData/Runner-btumkhqxnfgahyertjizuikcyuzl/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/geolocator_apple.build/Objects-normal/arm64/GeolocatorPlugin.o /Users/user1/development/tools/flutter/.pub-cache/hosted/pub.dartlang.org/geolocator_apple-2.2.0/ios/Classes/GeolocatorPlugin.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'geolocator_apple' from project 'Pods')
/Users/user1/development/tools/flutter/.pub-cache/hosted/pub.dartlang.org/connectivity_plus-2.3.5/ios/Classes/ReachabilityConnectivityProvider.swift:23:5: warning: result of call to 'ensureReachability()' is unused
        ensureReachability()
        ^                 ~~
/Users/user1/development/tools/flutter/.pub-cache/hosted/pub.dartlang.org/connectivity_plus-2.3.5/ios/Classes/PathMonitorConnectivityProvider.swift:31:5: warning: result of call to 'ensurePathMonitor()' is unused
        ensurePathMonitor()
        ^                ~~
/Users/user1/development/tools/flutter/.pub-cache/hosted/pub.dartlang.org/connectivity_plus-2.3.5/ios/Classes/PathMonitorConnectivityProvider.swift:35:5: warning: result of call to 'ensurePathMonitor()' is unused
        ensurePathMonitor()
        ^                ~~
/Users/user1/development/tools/flutter/.pub-cache/hosted/pub.dartlang.org/connectivity_plus-2.3.5/ios/Classes/PathMonitorConnectivityProvider.swift:31:5: warning: result of call to 'ensurePathMonitor()' is unused
        ensurePathMonitor()
        ^                ~~
/Users/user1/development/tools/flutter/.pub-cache/hosted/pub.dartlang.org/connectivity_plus-2.3.5/ios/Classes/PathMonitorConnectivityProvider.swift:35:5: warning: result of call to 'ensurePathMonitor()' is unused
        ensurePathMonitor()
        ^                ~~
/Users/user1/development/tools/flutter/.pub-cache/hosted/pub.dartlang.org/connectivity_plus-2.3.5/ios/Classes/ReachabilityConnectivityProvider.swift:23:5: warning: result of call to 'ensureReachability()' is unused
        ensureReachability()
        ^                 ~~
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:224:29: error: concurrency is only available in iOS 15.0.0 or newer
        open func downloadURL() async throws -> URL {
                                ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:293:25: error: concurrency is only available in iOS 15.0.0 or newer
        open func listAll() async throws -> StorageListResult {
                            ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:377:29: error: concurrency is only available in iOS 15.0.0 or newer
        open func getMetadata() async throws -> StorageMetadata {
                                ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:407:59: error: concurrency is only available in iOS 15.0.0 or newer
        open func updateMetadata(_ metadata: StorageMetadata) async throws -> StorageMetadata {
                                                              ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:441:24: error: concurrency is only available in iOS 15.0.0 or newer
        open func delete() async throws {
                           ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/user1/Library/Developer/Xcode/DerivedData/Runner-btumkhqxnfgahyertjizuikcyuzl/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/FirebaseStorage.build/Objects-normal/x86_64/StorageMetadata.dia:1:1: warning: Could not read serialized diagnostics file: error("Invalid diagnostics signature") (in target 'FirebaseStorage' from project 'Pods')
    Command CompileSwift failed with a nonzero exit code
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:224:29: error: concurrency is only available in iOS 15.0.0 or newer
        open func downloadURL() async throws -> URL {
                                ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:293:25: error: concurrency is only available in iOS 15.0.0 or newer
        open func listAll() async throws -> StorageListResult {
                            ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:377:29: error: concurrency is only available in iOS 15.0.0 or newer
        open func getMetadata() async throws -> StorageMetadata {
                                ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:407:59: error: concurrency is only available in iOS 15.0.0 or newer
        open func updateMetadata(_ metadata: StorageMetadata) async throws -> StorageMetadata {
                                                              ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:441:24: error: concurrency is only available in iOS 15.0.0 or newer
        open func delete() async throws {
                           ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/user1/Library/Developer/Xcode/DerivedData/Runner-btumkhqxnfgahyertjizuikcyuzl/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/FirebaseStorage.build/Objects-normal/x86_64/StorageObservableTask.dia:1:1: warning: Could not read serialized diagnostics file: error("Invalid diagnostics signature") (in target 'FirebaseStorage' from project 'Pods')
    Command CompileSwift failed with a nonzero exit code
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:224:29: error: concurrency is only available in iOS 15.0.0 or newer
        open func downloadURL() async throws -> URL {
                                ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:293:25: error: concurrency is only available in iOS 15.0.0 or newer
        open func listAll() async throws -> StorageListResult {
                            ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:377:29: error: concurrency is only available in iOS 15.0.0 or newer
        open func getMetadata() async throws -> StorageMetadata {
                                ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:407:59: error: concurrency is only available in iOS 15.0.0 or newer
        open func updateMetadata(_ metadata: StorageMetadata) async throws -> StorageMetadata {
                                                              ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:441:24: error: concurrency is only available in iOS 15.0.0 or newer
        open func delete() async throws {
                           ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:224:29: error: concurrency is only available in iOS 15.0.0 or newer
        open func downloadURL() async throws -> URL {
                                ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:293:25: error: concurrency is only available in iOS 15.0.0 or newer
        open func listAll() async throws -> StorageListResult {
                            ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:377:29: error: concurrency is only available in iOS 15.0.0 or newer
        open func getMetadata() async throws -> StorageMetadata {
                                ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:407:59: error: concurrency is only available in iOS 15.0.0 or newer
        open func updateMetadata(_ metadata: StorageMetadata) async throws -> StorageMetadata {
                                                              ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:441:24: error: concurrency is only available in iOS 15.0.0 or newer
        open func delete() async throws {
                           ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/user1/Library/Developer/Xcode/DerivedData/Runner-btumkhqxnfgahyertjizuikcyuzl/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/FirebaseStorage.build/Objects-normal/i386/StorageMetadata.dia:1:1: warning: Could not read serialized diagnostics file: error("Invalid diagnostics signature") (in target 'FirebaseStorage' from project 'Pods')
    Command CompileSwift failed with a nonzero exit code
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:407:59: error: concurrency is only available in iOS 15.0.0 or newer
        open func updateMetadata(_ metadata: StorageMetadata) async throws -> StorageMetadata {
                                                              ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:441:24: error: concurrency is only available in iOS 15.0.0 or newer
        open func delete() async throws {
                           ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/user1/Library/Developer/Xcode/DerivedData/Runner-btumkhqxnfgahyertjizuikcyuzl/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/FirebaseStorage.build/Objects-normal/arm64/StorageMetadata.dia:1:1: warning: Could not read serialized diagnostics file: error("Invalid diagnostics signature") (in target 'FirebaseStorage' from project 'Pods')
    Command CompileSwift failed with a nonzero exit code                                ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:407:59: error: concurrency is only available in iOS 15.0.0 or newer
        open func updateMetadata(_ metadata: StorageMetadata) async throws -> StorageMetadata {
                                                              ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:441:24: error: concurrency is only available in iOS 15.0.0 or newer
        open func delete() async throws {
                           ^
    /Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:25:39: note: add @available attribute to enclosing class
    @objc(FIRStorageReference) open class StorageReference: NSObject {
                                          ^
    Command CompileSwiftSources failed with a nonzero exit code
    note: Using new build system
    note: Planning
    note: Build preparation complete
    note: Building targets in parallel
    note: Removed stale file '/Users/user1/Library/Developer/Xcode/DerivedData/Runner-btumkhqxnfgahyertjizuikcyuzl/Build/Products/Debug-iphonesimulator/firebase_core/firebase_core.framework'
    /Users/projectPath/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'leveldb-library' from project 'Pods')
    /Users/projectPath/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'ReachabilitySwift' from project 'Pods')
    /Users/projectPath/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'MTBBarcodeScanner' from project 'Pods')
    /Users/projectPath/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'TOCropViewController-TOCropViewControllerBundle' from project 'Pods')
    /Users/projectPath/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'TOCropViewController' from project 'Pods')
    Result bundle written to path:
    	/var/folders/5m/btwjgwbs3zz4xt0v29h78lsm0000gn/T/flutter_tools.GaL813/flutter_ios_build_temp_dirDysItT/temporary_xcresult_bundle
Uncategorized (Xcode): Command CompileSwift failed with a nonzero exit code
Swift Compiler Error (Xcode): Concurrency is only available in iOS 15.0.0 or newer
/Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:223:28
Swift Compiler Error (Xcode): Concurrency is only available in iOS 15.0.0 or newer
/Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:292:24
Swift Compiler Error (Xcode): Concurrency is only available in iOS 15.0.0 or newer
/Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:376:28
Swift Compiler Error (Xcode): Concurrency is only available in iOS 15.0.0 or newer
/Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:406:58
Swift Compiler Error (Xcode): Concurrency is only available in iOS 15.0.0 or newer
/Users/projectPath/ios/Pods/FirebaseStorage/FirebaseStorage/Sources/StorageReference.swift:440:23
Uncategorized (Xcode): Command CompileSwiftSources failed with a nonzero exit code
Could not build the application for the simulator.
Error launching application on iPhone 12.
Exited
``

**Comment by stuartmorgan** [comment]
That error is coming from `FirebaseStorage`; issues with third -party plugins need to be reported to the developers of those plugins.
